cnf-tests/testsuites/validationsuite/cluster/validation.go

```diff
@@ -408,4 +411,84 @@ var _ = Describe("validation", func() {
 			Expect(mcpExist).To(BeTrue(), fmt.Sprintf("was not able to find the xt_u32 machine config %s in the %s machine config pool", mc.Name, mcp.Name))
 		})
 	})
+
+	Context("n3000", func() {
+		var n3000NodeList = &fpgav1.N3000NodeList{}
+
+		BeforeEach(func() {
+			testclient.Client.List(context.Background(), n3000NodeList)
+			if len(n3000NodeList.Items) < 1 {
```

Comment:
move this as an `It` if the operator is not deploy the test will fail here but it should fail in the `It`